docs/tutorial/certificates-autoenrolment.md

```diff
@@ -1,5 +1,185 @@
+    status: MONITORING
+    stuck: no
+    key pair storage: type=FILE,location='/var/lib/adsys/private/certs/galacticcafe-CA.Machine.key'
+    certificate: type=FILE,location='/var/lib/adsys/certs/galacticcafe-CA.Machine.crt'
+    CA: galacticcafe-CA
+    issuer: CN=galacticcafe-CA,DC=galacticcafe,DC=com
+...
+...
+Request ID 'galacticcafe-CA.Workstation':
+    status: MONITORING
+    stuck: no
+    key pair storage: type=FILE,location='/var/lib/adsys/private/certs/galacticcafe-CA.Workstation.key'
+    certificate: type=FILE,location='/var/lib/adsys/certs/galacticcafe-CA.Workstation.crt'
+    CA: galacticcafe-CA
+    issuer: CN=galacticcafe-CA,DC=galacticcafe,DC=com
+...
+...
+```
+
+From this truncated output we can see that there are two certificates being monitored:
+
+- `galactic-CA.Machine`
+- `galactic-CA.Workstation`
+
+These correspond to certificate templates that are configured on the certificate authority.
+
+The paths to the private key and certificate are included in the `getcert list` output.
+Everything should now be in place for the use of corporate services like VPNs and WiFi.
+
+## Connect to VPN server using certificates
+
+To check the VPN configuration run:
+
+```text
+cat /etc/ppp/peers/azure-vpn
+```
+
+Output:
+
+
+```text
+remotename: azure-vpn
+linkname: azure-vpn
+ipparamname: azure-vpn
+...
+...
+name        keypress.galacticcafe.com
+plugin      sstp-pppd-plugin.so
+...
+...
+ca: /var/lib/adsys/certs/galacticcafe-CA.2.crt
+cert: /var/lib/adsys/certs/galacticcafe-CA.Machine.crt
+key: /var/lib/adsys/private/certs/galacticcafe-CA.Machine.crt
+...
+...
+```
+
+An SSTP VPN is being used for this tutorial, connecting to a gateway in the Azure cloud.
+The name specified is the FQDN of the machine that the certificates are generated for.
+Confirm that paths to the `ca`, `cert` and private `key` are all specified.
+
+It should then be possible to connect to the VPN:
+
+```text
+sudo pon azure-vpn
+```
+
+Establishing the connection may take a few seconds.
+
+To check the connection run:
+
+```text
+ip a
+```
+
+This should output a point-to-point connection:
+
+```text
+...
+...
+8: ppp0: <POINTTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 pfifo_fast state unknown group default qlen 3
+...
+...
+
+```
+
+## Accessing resources on a virtual network
+
+The machine should now be connected to a virtual network with access to virtual resources.
+
+For example, if an Ubuntu machine has no public IP but is set up in the same virtual network then it should be accessible:
+
+```text
+ping <IPv4-address-of-resource>
+```
+
+It should be possible to `ssh` into a machine on the network:
+
+```text
+ssh -i ~/.ssh/adsys-integration.pem root@<IPv4-address-of-resource>
+```
+
+For example, an instance of Ubuntu Mantic Minotaur (development branch) will give an output that shows it is running on Azure based on the kernel version:
+
+```text
+Welcome to Ubuntu Mantic Minotaur (development branch) (GNU/Linux 6.5.0-1004-azure x86_64))
```

Comment:
Same here, replace by
```
Welcome to Ubuntu 24.04 LTS (GNU/Linux 6.5.0-1004-azure x86_64)
```

(the kernel version number is probably inaccurate but it is not important)

Reply:
done in [1270c01](https://github.com/ubuntu/adsys/pull/1073/commits/1270c01359b4840b6b57e6d06e7cfe0c867ab834)